src/components/diagrams/networkAreaDiagram/network-area-diagram-content.tsx

```diff
@@ -26,97 +26,103 @@ import { storeNetworkAreaDiagramNodeMovement } from '../../../redux/actions';
 import { PARAM_INIT_NAD_WITH_GEO_DATA } from '../../../utils/config-params';
 import { getNadIdentifier } from '../diagram-utils';
 import { UUID } from 'crypto';
+
+export function getValueFromThreshold(
+    value: number,
+    threshold: number,
+    aboveThreshold: string,
+    belowThreshold: string
+) {
+    return value > threshold ? aboveThreshold : belowThreshold;
+}
```

Comment:
you've exported this methods from commons-ui, no need to recreate it here

Reply:
removed